src/ast/passes/codegen_llvm.cpp

```diff
@@ -45,6 +45,10 @@
 #include "types.h"
 #include "usdt.h"
 
+#define _BT_DISASM_TMPFILE ".temp_bt_disassemble"
+static constexpr auto BT_DISASM_TMPFILE = _BT_DISASM_TMPFILE;
+static constexpr auto BT_DISASM_CMD = "llvm-objdump -d " _BT_DISASM_TMPFILE;
```

Comment:
Do we need to check if this is available on the host and perhaps generate a nicer error if it's not?

Reply:
Error seems reasonable enough IMO:
```
sudo PATH= ./build/src/bpftrace -e 'BEGIN { print(*curtask) }' -d dis -dcodegen-opt --test codegen 2>&1 | tail
!49 = distinct !DISubprogram(name: "BEGIN_1", linkageName: "BEGIN_1", scope: !2, file: !2, type: !50, flags: DIFlagPrototyped, spFlags: DISPFlagDefinition, unit: !46, retainedNodes: !53)
!50 = !DISubroutineType(types: !51)
!51 = !{!35, !52}
!52 = !DIDerivedType(tag: DW_TAG_pointer_type, baseType: !43, size: 64)
!53 = !{!54}
!54 = !DILocalVariable(name: "ctx", arg: 1, scope: !49, file: !2, type: !52)

Disassembled bytecode
---------------------------
sh: line 1: llvm-objdump: No such file or directory
```

Reply:
Eh... that's not such a graceful failure. Looks like a bug in our code TBH.

Reply:
Fair enough. But I'd also rather avoid overcomplicating something that'll have like N=3 users

Reply:
Kk, don't feel strongly enough and you're right, it's probably not a highly used feature.